spec/indexers/hyrax/valkyrie_work_indexer_spec.rb

```diff
@@ -21,4 +20,11 @@
 
     it_behaves_like 'a Basic metadata indexer'
   end
+
+  context 'when extending with custom metadata' do
```

Comment:
This context is the spec that is addressing the Actual Behavior vs. Expected Behavior described in the PR comments.